doc/api_package.rst

```diff
@@ -38,21 +42,57 @@
     commandline repository. The checksum represents @pkgid value which links primary metadata with
     other repository metadata files.
 
+  .. attribute:: conflicts
+
+    Packages that the package conflicts with (list of Hawkey.Reldep).
+
   .. attribute:: debug_name
 
     The name of the gebug-info package (string).
 
+  .. attribute:: description
+
+    The description of the package (string).
+
   .. attribute:: downloadsize
 
     The size of rpm package in bytes (integer).
 
   .. attribute:: epoch
 
-    Epoch of the package (integer)
+    Epoch of the package (integer).
+
+  .. attribute:: enhances
+
+    Packages that the package enhances (list of Hawkey.Reldep).
+
+  .. attribute:: evr
+
+    EVR (epoch:version-revision) of the package (string).
 
   .. attribute:: files
 
-    Files the package provides (list of strings)
+    Files the package provides (list of strings).
+
+  .. attribute:: group
+
+    Group of the package (string).
+
+  .. attribute:: hdr_chksum
+
+    Tuple with package header checksum and checksum type or None. The checksum is returned only for installed packages.
+
+  .. attribute:: hdr_end
+
+    Header end index for the package. Returns 0 for not known. (integer).
+
+  .. attribute:: changelogs
+
+    Changelogs for the package (list of dictionaries with "timestamp", "author" and "text" keys).
+
+  .. attribute:: installed
+
+    Returns True if the package is installed (boolean).
```

Comment:
`True` used to be marked with backticks.. There are some more type names on which backticks could be used in this PR too...